doc/source/calacs.rst

```diff
@@ -210,13 +213,11 @@ taken after January 2015.
 Pixel-Based CTE Correction (PCTECORR)
 -------------------------------------
 
-For all full-frame WFC exposures, pixel-based CTE correction (ACS ISR to be updated)
-is applied in ACSCTE that occurs after the ACSCCD series;
-i.e., after BLEVCORR.
+For all full-frame WFC exposures, pixel-based CTE correction (ACS ISR 2018-04)
```

Comment:
Should probably include link to ISR like in other instances in this documentation.

Reply:
Cool, I've added a link here and to other ISRs cited in this file.